src/themes/default/components/core/blocks/Microcart/Product.vue

```diff
@@ -95,17 +128,28 @@ import config from 'config'
 import Product from '@vue-storefront/core/compatibility/components/blocks/Microcart/Product'
 
 import ProductImage from 'theme/components/core/ProductImage'
+import ColorSelector from 'theme/components/core/ColorSelector.vue'
+import SizeSelector from 'theme/components/core/SizeSelector.vue'
 import RemoveButton from './RemoveButton'
 import BaseInputNumber from 'theme/components/core/blocks/Form/BaseInputNumber'
 import { onlineHelper } from '@vue-storefront/core/helpers'
+import { ProductOption } from '@vue-storefront/core/modules/catalog/components/ProductOption'
 
 export default {
   components: {
     RemoveButton,
     BaseInputNumber,
-    ProductImage
+    ProductImage,
+    ColorSelector,
+    SizeSelector
+  },
+  mixins: [Product, ProductOption],
+  beforeMount () {
```

Comment:
Would be cool to remove the event bus call - probably it requires a change to GenericSelector to call some event instead - but probably we should just keep it as it is and have it in mind with Product page refactor @filrak and @patzick - #3093

Reply:
Agree, but I think it's a separated task because would be nice to change it everywhere  _GenericSelector_ occurs and get rid of this event bus inside

Reply:
OK